plugins/csp-atmospheres/src/models/bruneton/internal/CSVLoader.hpp

```diff
@@ -0,0 +1,74 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////
+//                               This file is part of CosmoScout VR                               //
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+// SPDX-FileCopyrightText: German Aerospace Center (DLR) <cosmoscout@dlr.de>
+// SPDX-License-Identifier: MIT
+
+#ifndef CSP_ATMOSPHERES_MODELS_BRUNETON_INTERNAL_CSVLOADER_HPP
+#define CSP_ATMOSPHERES_MODELS_BRUNETON_INTERNAL_CSVLOADER_HPP
+
+#include <functional>
+#include <string>
+#include <vector>
+
+namespace csp::atmospheres::models::bruneton::internal {
+
+/// This static class is used to load the phase functions, scattering & absorption coefficients, as
+/// well as the density distributions of atmospheric particles from CSV files.
+class CSVLoader {
```

Comment:
A purely static class shouldn't exists. We should prefer a namespace with functions instead.